tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite.py

```diff
@@ -0,0 +1,115 @@
+import pathlib
+import great_expectations as gx
+import tempfile
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+
+from great_expectations.core.expectation_suite import ExpectationSuite
+
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_context">
+import great_expectations as gx
+
+context = gx.get_context()
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite create_validator">
+validator = context.sources.pandas_default.read_csv(
+    "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+)
+# </snippet>
+
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite add_2_expectations">
+validator.expect_column_values_to_not_be_null("pickup_datetime")
+validator.expect_column_values_to_be_between("passenger_count", auto=True)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_suite">
+my_suite = validator.get_expectation_suite()
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite show_suite">
+my_suite.show_expectations_by_expectation_type()
```

Comment:
This one is a litte trickier because the `show_expectations_by_expectation_type()` doesn't actually return anything, just uses pretty-print to output to console. Is there a good way to catch / test the output?

Reply:
Good point - how about something like this context manager approach? https://stackoverflow.com/a/16571630/2916375